utility.css

```diff
@@ -0,0 +1,137 @@
+.header__navigation {
```

Comment:
Have a look at your styles in the `utility.css` file. A lot of the styles here could be extracted as components and added to their own file :)